graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -65,3 +68,41 @@ def _get_edge_type_from_direct_edge(edge_name, direct_edge_descriptor):
     return EdgeType(edge_name, False, {}, {},
                     base_in_connection=direct_edge_descriptor.from_vertex,
                     base_out_connection=direct_edge_descriptor.to_vertex)
+
+
+def _get_sqlalchemy_indexes(vertex_name_to_table, vertex_types):
+    """Return the IndexDefinition objects corresponding to the indexes in the SQL tables."""
+    # TODO(pmantica): Add non-primary key indexes.
+    index_definitions = set()
+    for vertex_name, table in vertex_name_to_table.items():
+        vertex_type = vertex_types[vertex_name]
+        for constraint in table.constraints:
+            # When a primary key constraint is created in a SQL database, an accompanying
+            # primary key index is also created. However, SQLAlchemy explicitly does not include
+            # primary key indexes in table.indexes. Therefore, we use the PrimaryKeyConstraint
+            # to create the IndexDefinition. The only issue with doing so is that technically
+            # the index name is incorrect.
```

Comment:
Can you just not provide the name then? It's optional.

Reply:
Ok makes sense.